search-sentiments-map/public/scripts/map-script.js

```diff
@@ -97,6 +112,10 @@ function loadCountryData() {
     document.getElementById('data-max').textContent =
         dataMax.toLocaleString();
   });
+  const extremaHeader = document.getElementById('extrema-sentiment');
+  extremaHeader.innerText = 
+      'Most Negative Country: ' + countryMin + ', Most Positive Country: ' +
+      countryMax;
```

Comment:
Could we possibly swap these so positive comes first, then negative?

Reply:
It's arranged left to right, so I thought it would make most sense to do most negative then positive cuz the number line is -100 to 100

Reply:
It's not next to the legend though so I don't think it matters.

Reply:
+1 for positive first